packages/jaeger-ui/src/components/TracePage/TraceStatistics/tableValues.tsx

```diff
@@ -115,7 +115,7 @@ function valueFirstDropdown(selectedTagKey: string, trace: Trace) {
   const allSpans = trace.spans;
   // all possibilities that can be displayed
   if (selectedTagKey === serviceName) {
-    const temp = _.chain(allSpans)
+    const temp = _get.chain(allSpans)
```

Comment:
`chain` is not a function provided by `_get`, rather is a submodule itself. Please check the comments above ^